test/cropharvest/eo/test_eo.py

```diff
@@ -37,8 +39,8 @@ def test_labels_to_polygons_and_years(with_identifier, monkeypatch):
         assert identifier == "hello_world"
     else:
         expected_identifier = (
-            "min_lat=45.2202_min_lon=-75.5529_max_lat=47.2202_"
-            + "max_lon=-73.5529_dates=2020-12-15_2021-12-15_all"
+            "min_lat=46.2195_min_lon=-74.5539_max_lat=46.221_"
+            + "max_lon=-74.5518_dates=2020-12-15_2021-12-15_all"
```

Comment:
Using the real bounding_box_from_centre results in different size bbox so test has to be updated